pdr_backend/subgraph/info725.py

```diff
@@ -90,3 +92,21 @@ def info725_to_info(info725: list) -> Dict[str, Optional[str]]:
                 break
 
     return info
+
+
+def get_pair_timeframe_source_from_contract(contract):
+    if contract["token"]["nft"]:
+        info725 = contract["token"]["nft"]["nftData"]
+        info = info725_to_info(info725)  # {"pair": "ETH/USDT", }
+        pair = info["pair"]
+        timeframe = info["timeframe"]
+        source = info["source"]
+        return (pair, timeframe, source)
+
+    if contract["id"] in WHITELIST_FEEDS_MAINNET:
```

Comment:
i have a feeling this will be the result 99% of the time, but i get why you made this choice